### PR TITLE
Make newsletter subscription changes less confusing (Fixes #8112)

### LINKED ISSUE #8112: Subscriber Feedback: "Email preferences don't make sense"
> Hello friends,
> I dunno if this is corretc place to repurt bug, but I am confusion of the email preferences.
> Is it yes to subscibe, no to unsubscribe? Please help me lol.
>
> screenshot

Original issue filed here: https://github.com/mozilla/foundation.mozilla.org/issues/3792

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Make the form less confusing.
- Fix page layout on small screens.

## Issue / Bugzilla link
#8112

### CODE REVIEW

File: bedrock/newsletter/templates/newsletter/existing.html [hunk 1]
```diff
@@ -114,7 +114,6 @@ <h1>{{ _('Manage your Email Preferences') }}</h1>
           <tr>
             <th></th>
             <th>{{ _('Subscribe') }}</th>
-            <th>{{ _('Unsubscribe') }}</th>
```
Comment (pmac) [inline bedrock/newsletter/templates/newsletter/existing.html]: There are still 3 columns to this table. Do we need a `colspan=2` on the `<th>` above?

Comment (alexgibson) [reply bedrock/newsletter/templates/newsletter/existing.html]: Good catch 👍

**Comment by alexgibson** [comment]
Thanks @pmac updated

**Comment by pmac** [comment]
Had to dust off the old `<table>` areas of the brain... just more evidence of getting old 😄

**Pull request merged**